canvas_account_admin_tools/settings/aws.py

```diff
@@ -20,3 +20,5 @@
 SESSION_COOKIE_SECURE = True
 
 dictConfig(LOGGING)
+
+CONCLUDE_COURSES_URL = SECURE_SETTINGS['conclude_courses_url']
```

Comment:
I would move this up a bit.  Pretty sure we want the last thing in this settings file to be the `dictConfig` line.